entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+cd web
+
+# Run migration only if explicitly set via ENV
+if [ "$RUN_MIGRATION" != "false" ]; then
+	./manage.py migrate
+fi
+
+if [ "$ASSING_LOCATIONS" != "false" ]; then
+	./manage.py pa_find_locations
+fi
+
+# don't create an admin interface per default
+#./manage.py createsuperuser  
+
+# start the server
+./manage.py $@
```

Comment:
You might want to use `exec` here.

> […] If we call `prog1 -> prog2 -> prog3 -> prog4` etc. and never go back, then make each call an exec. It saves resources (not much, admittedly, unless repeated) and makes shutdown simplier.

[StackOverflow explainer about `exec`'s purposes](https://stackoverflow.com/a/18351547/1072129)

Reply:
After `entrypoint.sh` is run, the container CMD should be executed. IMHO `exec` would prevent this(?)

Reply:
I think we have a misunderstanding here. I referred to the `exec` Bash built-in. How would it prevent running the command specified in `CMD`?